src/Client/app/(admin)/exams/create/page.tsx

```
@@ -314,171 +304,182 @@ useEffect(() => {
+                                value={date}
+                                onChange={setDate}
+                            />
+                            <Input
+                                className="flex-1 rounded-2xl"
+                                isRequired
+                                label="Duration (minutes)"
+                                type="number"
+                                min="1"
+                                value={formData.durationMinutes.toString()}
+                                onChange={(e) => {
+                                    const value = parseInt(e.target.value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)